src/ray/gcs/tables.cc

```diff
@@ -1,7 +1,65 @@
 #include "ray/gcs/tables.h"
 
+#include "ray/gcs/client.h"
+
+#include "task.h"
+#include "common_protocol.h"
+
 namespace ray {
 
-namespace gcs {}  // namespace gcs
+namespace gcs {
+
+std::shared_ptr<TaskTableDataT> MakeTaskTableData(
```

Comment:
It seems these 3 functions are file-local helpers.  If that's true, the better practice is to wrap them in an anonymous `namespace { }` (decls in .h, defs in .cc still).

Reply:
Only one of them is file local, I put it into a anonymous namespace :)